lib/index.ts

```diff
@@ -19,52 +19,65 @@ var XML2JS_OPTS = {
  * @param {object} opt Is the option with the filePath or xmlContent and the optional format.
  * @return {object} The pom object along with the timers.
  */
-function parse(opt, callback) {
+function parse(
+  opt: (Options & { filePath?: string; xmlContent?: string }) | null,
+  callback: (
+    e: Error | null,
+    r?: {
+      pomXml: string;
+      pomObject: object;
+    } | null
+  ) => void
+) {
   if (!opt) {
-    throw new Error("You must provide options: opt.filePath and any other option of " +
-      "https://github.com/Leonidas-from-XIV/node-xml2js#options");
+    throw new Error(
```

Comment:
Please exclude pure formatting changes from the PR.

Reply:
I tried to keep them away but they still crept in my bad! Removed these changes.

Reply:
Thanks.